TestApps/SIL.Email.TestApp/SIL.Email.TestApp.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```

Comment:
I don't think this project needs it. (Notice there is no Nunit package reference)